chaos_genius/core/utils/string_helpers.py

```diff
@@ -7,11 +7,34 @@
 
 # TODO: Update docstrings to sphinx format
 
+# Added a string translation to ensure we escape double quotes correctly.
+# Replaces all '"' with '\"'
+_string_transformation_for_query = str.maketrans({'"': '\\"'})
 
-def _parse_single_col_for_query_string(col: str, value: str) -> str:
+
+def get_query_from_col_and_value(col: str, value: str) -> str:
+    """Convert given col and value into a query string."""
+    col = col.translate(_string_transformation_for_query)
+    value = value.translate(_string_transformation_for_query)
     return f'`{col}`=="{value}"'
 
 
+def get_query_from_cols_and_values(col_names, raw_combinations):
+    """Convert given col names and combinations into query strings."""
+    # TODO: Write tests for this.
+    query_list = []
+    for comb in raw_combinations:
+        if type(comb) == str:
+            unjoined_query = [get_query_from_col_and_value(col_names[0], comb)]
+        elif type(comb) == tuple:
+            unjoined_query = [
+                get_query_from_col_and_value(col_names[i], comb[i]) for i in range(len(comb))
```

Comment:
The else case is not handled here. This will raise an error saying `unjoined_query` is unbound. Would be better to raise a more descriptive error explicitly.